net-p2p/go-ethereum/go-ethereum-1.8.2.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
```

Comment:
But… this is still combining removal with adding new version?

Reply:
I thought you meant only 1.7.2 needed to be removed separately, changed.